### ISSUE TITLE
Bootstrap 3.0 will no longer have typeahead support.

### ISSUE THREAD

**Issue by blimey85** [soliantconsulting/tagmanager#6]
I was using the tagmanager that you originally based yours off of and then ran into an issue when I tried to swap out Bootstrap Typeahead for Twitters typeahead.js. Yesterday it was announced that Bootstrap is dropping their native typeahead functionality.

<blockquote class="twitter-tweet" data-cards="hidden"><p>Heads up on v3 change: we're dropping our typeahead for @<a href="https://twitter.com/twitter">twitter</a>'s typeahead.js. Relevant commit: <a href="https://t.co/3PpUvl7J12" title="https://github.com/twitter/bootstrap/commit/88dd9de9bad56b30a12466477a810c78648200c9">github.com/twitter/bootst…</a></p>&mdash; Twitter Bootstrap (@twbootstrap) <a href="https://twitter.com/twbootstrap/status/332334721029791745">May 9, 2013</a></blockquote>

<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>

I'm going to try setting yours up in my project and see how it plays with Twitter's typeahead.js and then I'll report back.

**Comment by blimey85** [comment]
I dropped in your tag manager overwriting the previous version and then in my config I changed `source:` to `local:` and it worked. I'm using it with Bootstrap and I'm going to have to do some cleanup of the css to get everything playing nicely in my setup but that may not be necessary for other people.

Anyway, going forward the typeahead library to use with Bootstrap will be: https://github.com/twitter/typeahead.js. That's the one they are recommending and it seems to work quite well. Very similar to the one included with Bootstrap but better supported and with more features.
